web-components/src/components/modal/wallet-modal/components/WalletModal.Mobile.config.tsx

```diff
@@ -0,0 +1,46 @@
+import QRCode from 'qrcode.react';
+
+import { Center } from '../../../../components/box';
+import ContainedButton from '../../../../components/buttons/ContainedButton';
+import { IconTabProps } from '../../../../components/tabs/IconTab';
+import { Subtitle } from '../../../../components/typography';
+
+type getWalletModalMobileTabsParams = {
+  mobileConnectUrl?: string;
+  qrCodeUri?: string;
+};
+
+export const getWalletModalMobileTabs = ({
+  mobileConnectUrl,
+  qrCodeUri,
+}: getWalletModalMobileTabsParams): IconTabProps[] =>
+  [
+    {
+      label: 'Mobile',
+      content: (
+        <Center sx={{ pt: 9, flexDirection: 'column', height: 340 }}>
+          <Subtitle sx={{ color: 'info.dark' }}>
+            {'Connect to Mobile Wallet'}
+          </Subtitle>
+          <Center sx={{ width: 300, height: 300 }}>
+            <a href={mobileConnectUrl}>
+              <ContainedButton>{'Connect'}</ContainedButton>
+            </a>
+          </Center>
+        </Center>
+      ),
+    },
+    {
+      label: 'Scan with the Keplr mobile app',
+      content: (
+        <Center sx={{ pt: 9, height: 340 }}>
+          <QRCode size={300} value={qrCodeUri ?? ''} />
+        </Center>
+      ),
+    },
```

Comment:
Actually, I mirrored Osmosis UX here. I think the idea is that it should be possible for example to connect from iPad while having the wallet only on a phone.
In any case, we always try to open the deep link by default, so the QR code on mobile will only be visible if the user is on a mobile device with no wallet installed.

Reply:
> In any case, we always try to open the deep link by default, so the QR code on mobile will only be visible if the user is on a mobile device with no wallet installed. 

I had Keplr mobile app installed and even if it tried to open the deep link by default indeed, I could cancel that and go to the "Scan with the Keplr mobile app" tab.

> Actually, I mirrored Osmosis UX here. I think the idea is that it should be possible for example to connect from a tablet while having the wallet only on a phone.

I then tried to scan the QR code using a second phone with Keplr app on it but that wasn't successful : once I scanned the QR code from the app, I could see the loading spinner for some time then it just disappeared but nothing else happened.

Reply:
@blushi I removed the QR code on mobile since it's not really working and it might have been a weird case, to begin with.
I redeployed the sandbox env, if you want to run some final tests: https://alike-spring.surge.sh/